examples/arpscan/arpscan.go

```diff
@@ -178,9 +178,8 @@ func ips(n *net.IPNet) (out []net.IP) {
 	mask := binary.BigEndian.Uint32([]byte(n.Mask))
 	net := num & mask
 	broadcast := net | ^mask
-	for net++; net < broadcast-1; {
+	for net++; net < broadcast-1; net++ {
```

Comment:
given this new approach, I don't think you want "broadcast - 1" anymore; you won't hit .254 in a /24 with this code.

Reply:
Fixed it!